UNITY.md

```diff
@@ -113,60 +111,57 @@ Each `<Placement>` in `PlacementRoot` is an empty GameObject. PlacementRoot is t
-        <!-- <Axis rotation="(0.0, 1.0, 0.0)" angle="0.0" /> -->
+        <!-- The object is in front of the origin (z axis flipped) -->
         <Position>(0.0, 0.0, -4.0)</Position>
         <LookAt target="(0.0, 0.0, 0.0)" up="(0.0, 1.0, 0.0)"/>
     </Placement>
     <Placement name="LeftWall">
         <RelativeTo>Center</RelativeTo>
           <!-- Look at the origin with the y axis pointed upwards -->
           <!-- The object is to the left of the origin so it must rotate to do so -->
-          <!-- <Axis rotation="(0.0, 1.0, 0.0)" angle="270.0" /> -->
         <Position>(-4.0, 0.0, 0.0)</Position>
         <LookAt target="(0.0, 0.0, 0.0)" up="(0.0, 1.0, 0.0)"/>
     </Placement>
     <Placement name="RightWall">
         <RelativeTo>Center</RelativeTo>
         <!-- Look at the origin with the y axis pointed upwards -->
         <!-- The object is to the right of the origin so it must rotate to do so -->
-        <!-- <Axis rotation="(0.0, 0.0, 0.0)" angle="90.0" /> -->
         <Position>(4.0, 0.0, 0.0)</Position>
         <LookAt target="(0.0, 0.0, 0.0)" up="(0.0, 1.0, 0.0)"/>
     </Placement>
     <Placement name="FloorWall">
         <RelativeTo>Center</RelativeTo>
         <!-- Look at the origin with the z axis pointed upwards -->
-        <!-- The object is below the origin so it must rotate to do so. Note that by rotating over the X axis the Z axis now points downward. -->
-        <!-- <Axis rotation="(1.0, 0.0, 0.0)" angle="90.0" /> -->
+        <!-- The object is below the origin so it must rotate to do so. Note that the Z axis points downward as a result of the rotation. -->
         <Position>(0.0, -4.0, 0.0)</Position>
         <LookAt target="(0.0, 0.0, 0.0)" up="(0.0, 0.0, -1.0)"/>
     </Placement>
 </PlacementRoot>
 ```
 
+Note that even though the `<PlacementRoot>` is identical between projects each wall is still instantiated from `CLI.cs`.
+
+### Placement
+
 - `RelativeTo`: Which wall inside `PlacementRoot` the object is a child of in the hierarchy  
   - Objects that are `<RelativeTo>Center</RelativeTo>` are children of `Story`.
-- `Position`: Local Position
-- [Axis || LookAt || Normal]
-  - `Axis`: The objects rotation around a specified axis
+- `<Position>`: The local position of the object
+- Rotation is an `<xs:choice>` (Axis, LookAt, Normal)
+  - `<Axis>`: The objects rotation around a specified axis
     - `rotation`: The axis around which the rotation takes place
     - `angle`: The rotation angle of the object, in degrees
-  - `LookAt`: Object is rotated to look at a specified point (in `Story` space)
+  - `<LookAt>`: Object is rotated to look at a specified point (in `Story` space)
     - `target`: The position the object is looking at
     - `up`: Which direction (world space) is considered up (+y)
-  - `Normal`: *Same as LookAt but with a normalized vector?*
+  - `<Normal>`: **TODO (63):** *Same as LookAt but with a normalized vector?*
```

Comment:
I think maybe if you have this as #63 it will automatically link it to the issue?

Reply:
Hmm it doesn't seem to? I wonder if it's because this is a generic markdown file and not README.md? I could write it as a web link but I think that's more work than it's worth

Reply:
Ah oh well D; now we know !